runtime/src/lib.rs

```diff
@@ -254,6 +258,63 @@ impl pallet_node_authorization::Config for Runtime {
     type WeightInfo = (); // no benchmarks defined in pallet!
 }
 
+parameter_types! {
+    pub const ProxyDepositBase: Balance = 0; // 0 indicates no cost to creating proxies - may be prone to spam
+    pub const ProxyDepositFactor: Balance = 0;
+    pub const AnnouncementDepositBase: Balance = 0;
+    pub const AnnouncementDepositFactor: Balance = 0;
+
+}
+
+/// The type used to represent the kinds of proxying allowed.
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, RuntimeDebug, MaxEncodedLen, TypeInfo)]
+pub enum ProxyType {
+    RunProcess,
+    Any,
+    Governance,
+}
+impl Default for ProxyType {
+    fn default() -> Self {
+        Self::Any
+    }
+}
+
+// Determines which calls a proxy type can execute.
+impl InstanceFilter<RuntimeCall> for ProxyType {
+    fn filter(&self, c: &RuntimeCall) -> bool {
+        match self {
+            ProxyType::Any => true,
+            ProxyType::RunProcess => matches!(c, RuntimeCall::UtxoNFT(..)),
+            ProxyType::Governance => matches!(c, RuntimeCall::TechnicalCommittee(..)),
+        }
+    }
+    fn is_superset(&self, o: &Self) -> bool {
+        match (self, o) {
+            (x, y) if x == y => true,
+            (ProxyType::Any, _) => true,  // `Any` is a superset of all
+            (_, ProxyType::Any) => false, // Nothing is a superset of `Any`
+            (ProxyType::RunProcess, ProxyType::Governance) => false, // RunProcess can't execute Governance calls
+            (ProxyType::Governance, ProxyType::RunProcess) => false, // Governance can't execute RunProcess calls
```

Comment:
I think these three lines are redundent? Surely the fallthough to `false` for `_` will also catch these